### ISSUE TITLE
Event order differs for platforms on orientation change

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#317]
Retrieving `screen.width` and `screen.height` in an `change:bounds` callback after rotating the device return different values on both platforms. On Android the updated `screen.width` and `screen.height`  values get returned, while on iOS they are still the ones valid before the rotation.

**Comment by jordiboehmelopez** [comment]
Can you provide a small snippet to reproduce?

**Comment by ralfstx** [comment]
```js
page.on("change:bounds", function() {
  console.log(screen.width, "x", screen.height);
});
```

Question is which behavior is right. Currently, the `change:bounds` is fired before the orientation change. At this point, page bounds and screen bounds are inconsistent. I know it was my suggestion that the `orientationchange` event should fire when the orientation change is finished, but I was not aware of this implication.

If we changed the order to fire `orientationchange` before `change:bounds`, there would be no inconsistency in the `change:bounds` event, but an in a `change:orientation` listener the widget bounds would not yet be adjusted to the new screen bounds.

**Comment by mpost** [comment]
We discussed previously that the orientation change callback is not the sufficient place to adjust a layout. Therefore i would expect that `change:orientation` is fired before the `change:bounds`.

**Comment by tbuschto** [comment]
I suppose it's more logical to change the orientation and screen properties before the relayout happens. I previously thought it would make sense to do so only after the resize animation is done, but in actuality the device itself is already re-orientated before this animation plays. Otherwise it would be the ui orientation property.

**Comment by ralfstx** [comment]
I agree that `change:orientation` should come before `change:bounds`.
